src/riak_kv_coverage_filter.erl

```diff
@@ -150,26 +154,26 @@ build_item_filter(FilterInput) ->
 
 
 %% @private
-build_preflist_fun(Bucket, CHBin) ->
+build_preflist_fun(Bucket, CHBin, KeyConvFn) ->
     fun({o, Key, _Value}) -> %% $ index return_body
-            ChashKey = riak_core_util:chash_key({Bucket, Key}),
+            ChashKey = riak_core_util:chash_key({Bucket, KeyConvFn(Key)}),
             chashbin:responsible_index(ChashKey, CHBin);
        ({_Value, Key}) ->
-            ChashKey = riak_core_util:chash_key({Bucket, Key}),
+            ChashKey = riak_core_util:chash_key({Bucket, KeyConvFn(Key)}),
```

Comment:
Yes, this is true.  I modified this code before I remembered that {o, Key, _Value} was a TS-specific pattern match, but left it in generic hook to allow transformations on any type of key.  

However, one thing we do need to think about is whether we are allowing mixed TS and non-TS keys to be written to a TS bucket, and what happens when we do.  In that case, we may actually need to differentiate between types of keys encountered in the same bucket, and only do transforms on one of them.

I'll investigate what happens if I try to write both types of keys to a TS bucket... that will inform what we do here.